package.json

```diff
@@ -29,7 +31,7 @@
   },
   "dependencies": {
     "bufrw": "^1.2.1",
-    "crc": "^3.2.1",
+    "crc": "3.2.1",
```

Comment:
It broke on master, now I'm pinning it because upstream broke compat in patch version.